test/language/expressions/arrow-function/static-init-await-binding.js

```diff
@@ -0,0 +1,21 @@
+// Copyright (C) 2021 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-identifiers
+description: The restriction on `await` as a BindingIdentifier does not cross function boundaries
+info: |
+  BindingIdentifier : Identifier
+
+  [...]
+  - It is a Syntax Error if the code matched by this production is nested,
```

Comment:
As of https://github.com/tc39/proposal-class-static-block/pull/46, this is changing such that the block is parsed in `+Await` and we issue an error if `ContainsAwait` returns **true**, so this section has been removed.